### ISSUE TITLE
Add Project : Ping Pong Game

### ISSUE THREAD

**Issue by robin025** [Vishal-raj-1/Awesome-JavaScript-Projects#1379]
# Is your feature request related to a problem? Please describe.
No, it's not a problem.

# Describe the solution you'd like
I would like to add a Ping Pong Game using JavaScript. Please assign me this issue under GSSOC'21 @ridsuteri @991rajat @harshgupta20

**Comment by ridsuteri** [comment]
@robin025 the project by the name of 'color picker and converter' does this job of converting rgb to hex .
you can improve it to add rgba to hex within that project , as adding a entirely new project simply for 'a' factor is not feasible according to me

**Comment by ridsuteri** [comment]
@robin025 if you are interested in improving that color converter project you can open another issue for that :)

**Comment by robin025** [comment]
i will be creating a color converter which will include all the color converters like RGB to HEX,CMYK,HSV,HSL and vice versa
will that be okay @ridsuteri  ?

**Comment by ridsuteri** [comment]
yeah this sounds great @robin025 and make changes directly to  hex to rgb converter since hex to rgb conversion feature is already available in the color picker and converter project
